go.mod

```diff
@@ -8,9 +8,13 @@ require (
 	github.com/hashicorp/go-argmapper v0.0.0-20200721221215-04ae500ede3b
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-tfe v0.11.1
-	github.com/hashicorp/terraform-plugin-go v0.2.1
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.3.0
-	github.com/nullstone-io/module v0.2.3
-	github.com/stretchr/testify v1.5.1
-	gopkg.in/nullstone-io/go-api-client.v0 v0.0.0-20210818163456-ae44a6533eb6
+	github.com/hashicorp/terraform-plugin-go v0.7.1
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
+	github.com/nullstone-io/module v0.2.4
+	github.com/stretchr/testify v1.7.0
+	gopkg.in/nullstone-io/go-api-client.v0 v0.0.0-20220221231733-8058a6eea44b
+	gopkg.in/nullstone-io/nullstone.v0 v0.0.43
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
+
+replace gopkg.in/nullstone-io/nullstone.v0 => gopkg.in/nullstone-io/nullstone.v0 v0.0.44-0.20220221233056-43f14e8cc51b
```

Comment:
Did you mean to leave this in here?

Reply:
Good find. I will drop this as soon as I merge and publish the Nullstone CLI.